Branch-SDK/src/main/java/io/branch/referral/DeviceInfo.java

```diff
@@ -286,15 +287,41 @@ public String getOsName() {
         return systemObserver_.getOS(context_);
     }
 
-    // PRS : User agent is checked only from api-17
+
+    /**
+     * Returns the browser's user agent string
+     * PRS : User agent is checked only from api-17
+     * @param context
+     * @return user agent string
+     */
     private String getDefaultBrowserAgent(Context context) {
         String userAgent = "";
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
@gdeluna-branch  : Is this version check needed at the top now that `getDefaultUserAgent` is moved down and `getUserAgentString` is available for all API versions and min API version that we support is 21+